src/tracks/InstrumentTrack.cpp

```diff
@@ -1130,6 +1156,37 @@ void InstrumentTrackView::midiConfigChanged()
 
 
 
+QMenu * InstrumentTrackView::createFxMenu(QString title, QString newFxLabel)
+{
+	int channelIndex = model()->effectChannelModel()->value();
+
+	FxChannel *fxChannel = Engine::fxMixer()->effectChannel( channelIndex );
+
+	QMenu *fxMenu = new QMenu( title.arg( channelIndex ).arg( fxChannel->m_name ) );
```

Comment:
I wanted the title variable to be interpolate parameters optionally. So that if I passed `%1` inside the string or not would not be much of a difference.